### ISSUE TITLE
pwsh arguments not recognized

### ISSUE THREAD

**Issue by souchprod** [PowerShell/PowerShell#11073]
# Steps to reproduce

```
pwsh -NoLogo
```

# Expected behavior

```
Prompt displayed without the version and copyright information
```

# Actual behavior

```
-NoLogo : The term '-NoLogo' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ -NoLogo
+ ~~~~~~~
+ CategoryInfo          : ObjectNotFound: (-NoLogo:String) [], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException
```

# Environment data

```
Name                           Value
----                           -----
PSVersion                      6.2.3
PSEdition                      Core
GitCommitId                    6.2.3
OS                             Microsoft Windows 10.0.19023
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

I have this issue with all the arguments i tested (-Interactive -NonInteractive -Help -Command=, and tested on both Windows and Linux (Centos).

Powershell core installed used:
```
dotnet tool install --global PowerShell
```

Is it a normal behavior ? I can't find any issue or documentation about it, but there is trace of usage like pwsh -Command everywhere. 

This is very blocking for me

**Comment by vexx32** [comment]
I believe @adityapatwardhan was working on the pwsh-as-a-global-tool aspect, perhaps he can provide some clarity here. 🙂

**Comment by mklement0** [comment]
Just to confirm: `pwsh -nologo` works just fine in a regularly installed version.

A quick experiment seems to indicate that a `dotnet tool install --global PowerShell`-installed version doesn't seem to recognize _any_ CLI parameters.

**Comment by souchprod** [comment]
Thanks @mklement0 , you saved my day!
Installing it regularly is a challenged, [Ms documentation](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6#centos-7), does not work, i had to use [the trick described in this issue](https://github.com/PowerShell/PowerShell/issues/8879) to succeed.

**Comment by adityapatwardhan** [comment]
@souchprod  - This should be fixed in the 7.0.0-rc.1 release of the PowerShell global tool. Can you try it again?

Installation: `dotnet tool install --global PowerShell --version 7.0.0-rc.1`

**Comment by adityapatwardhan** [comment]
This was fixed in 7.0.0-RC.1, please re-active the issue if you still see issues.

```
C:\>pwsh -nologo
PS C:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.0-rc.1
PSEdition                      Core
GitCommitId                    7.0.0-rc.1
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
